ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/tacacs_servers.schema.yml

```diff
@@ -9,6 +9,10 @@ keys:
   tacacs_servers:
     type: dict
     keys:
+      timeout:
+        type: int
+        convert_types:
+          - str
```

Comment:
If the min and max values are constant across platforms, please add them in the schema as well as a description indicating that this is in "seconds"
```
leaf1(config)#tacacs-server timeout ?
  <1-1000>  Number of seconds
```

Reply:
If I'll put a max/min for global timeout, I shall make the same for the timeout per host. Can I make it as part of the same PR?